src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/RolesController.kt

```diff
@@ -114,7 +114,7 @@ class RolesController(
         content = [
           Content(
             mediaType = "application/json",
-            array = ArraySchema(schema = Schema(implementation = RolesPaged::class))
+            array = ArraySchema(schema = Schema(implementation = Role::class))
```

Comment:
Should this be a "pagedresponse" ?

Reply:
I think the swagger was incorrect on this function before - it returns List<Role>, so not a paged response